docs/demo_guide/README.md

```diff
@@ -30,7 +30,7 @@ Setup steps:
     `Note:` If you do not have a GPU, you can use the following script instead
 
     ```
-    bash apollo.sh build_cpu
+    bash apollo.sh build_gpu
```

Comment:
It seems like incorrect, as "you do not have a GPU".

Reply:
agree. closed for now.